docs/resources/Channel.md

```diff
@@ -301,13 +301,13 @@ Represents a message sent in a channel within Discord.
 | resolved?                   | [resolved](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure) data                                   | data for users, members, channels, and roles in the message's [auto-populated select menus](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menus)                                                                                                                         |
-| poll?                       | [poll](#DOCS_RESOURCES_POLL/poll-object) object                                                                                           | A poll!                                                                                                                                                                                                                                                                 |
+| poll?\*\*                   | [poll](#DOCS_RESOURCES_POLL/poll-object) object                                                                                           | A poll!                                                                                                                                                                                                                                                                 |
 | call?                       | [message call](#DOCS_RESOURCES_CHANNEL/message-call-object) object                                                                        | the call associated with the message                                                                                                                                                                                                                                    |
 
 
 \* The author object follows the structure of the user object, but is only a valid user in the case where the message is generated by a user or bot user. If the message is generated by a webhook, the author object corresponds to the webhook's id, username, and avatar. You can tell if a message is generated by a webhook by checking for the `webhook_id` on the message object.
 
-\*\* An app will receive empty values in the `content`, `embeds`, `attachments`, and `components` fields if they have not configured (or been approved for) the [`MESSAGE_CONTENT` privileged intent (`1 << 15`)](#DOCS_TOPICS_GATEWAY/message-content-intent).
+\*\* An app will receive empty values in the `content`, `embeds`, `attachments`, `components`, and `poll` fields if they have not configured (or been approved for) the [`MESSAGE_CONTENT` privileged intent (`1 << 15`)](#DOCS_TOPICS_GATEWAY/message-content-intent).
```

Comment:
so im all for documenting this behavior more explicitly, but when i was testing it looked like polls was a bit different—`content`, `embeds`, and `attachments` are empty values, but `poll` is actually just not sent as a field when you don't have message content intent enabled.

would love for this to be consistent at API layer at some point and have it actually just be an empty object or a partial, but for now we should probably document explicitly that it isn't passed if the app doesn't have message content intent

Reply:
Oh well, I just intended to add the bits #6746 missed, but it seems like I opened a (small) can of worms here. Guess I'll think of a way to convey this as little verbose as possible these days / am open to suggestions in the meantime...